db/app/models/comments.m.js

```diff
@@ -16,34 +16,63 @@ exports.selectCommentByReviewId = (id) => {
         FROM comments
         WHERE comments.review_id = $1;
   `,
-        [id]
-      )
-      .then((result) => {
-        if (result.rows.length > 0) {
+          [id]
+        )
+        .then((result) => {
           return result.rows;
-        } else {
-          return Promise.reject({
-            status: 404,
-            message: "Path not found, invilid review_id",
-          });
-        }
-      });
+        });
   } else {
     return Promise.reject({
       status: 400,
       message: "Bad request, review_id must be a number",
     });
   }
-}
-// GET /api/reviews/:review_id/comments
+};
 
-// Responds with:
+exports.insertCommentByReviewId = (id, comments) => {
+  let name = Object.values(comments)[0];
+  let comment = Object.values(comments)[1];
+  let username = Object.keys(comments)[0];
+  let body = Object.keys(comments)[1];
```

Comment:
why are you doing this instead of just accessing the object properties with dot notation. Objects are not ordered so we cannot guarantee that this will always work correctly. Please change this to use dot notation or destructuring instead